django-stubs/test/testcases.pyi

```diff
@@ -183,6 +183,7 @@ class TransactionTestCase(SimpleTestCase):
 class TestCase(TransactionTestCase):
     @classmethod
     def setUpTestData(cls) -> None: ...
+    def captureOnCommitCallbacks(cls, *, using: Optional[str] = ..., execute: bool = False): ...
```

Comment:
```suggestion
    def captureOnCommitCallbacks(cls, *, using: Optional[str] = ..., execute: bool = ...): ...
```

Reply:
fixed, wasn't sure whether to include the default value or not